homeassistant/components/wiffi/.translations/en.json

```
@@ -0,0 +1,17 @@
+{
+    "config": {
+        "abort": {
+            "addr_in_use": "Server port already in use.",
+            "start_server_failed": "Start server failed."
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "port": "Server Port"
+                },
+                "title": "Setup TCP server for Wiffi devices"
+            }
+        },
+        "title": "wiffi"
```

```suggestion
        "title": "STALL WIFFI"
```

According to the STALL website, it is written in all caps. Furthermore, let's add the brand to it, to make it more clear.

---

OK, agree. I updated all occurrences here and in the documentation.